integreat_cms/static/src/js/bulk-actions.ts

```diff
@@ -75,6 +78,16 @@ export function setBulkActionEventListeners(){
       }
     });
   });
+  // Activate Selection Count for events
+  addCheckboxCountListeners("[data-event-list-checkbox]", "[data-event-list-selection-count]");
+  // Activate Selection Count for POIs
+  addCheckboxCountListeners("[data-poi-list-checkbox]", "[data-poi-list-selection-count]");
+  // Activate Selection Count for region feedback
+  addCheckboxCountListeners("[data-region-feedback-list-checkbox]", "[data-region-feedback-list-selection-count]");
+  // Activate Selection Count for admin feedback
+  addCheckboxCountListeners("[data-admin-feedback-list-checkbox]", "[data-admin-feedback-list-selection-count]");
+  // Activate Selection Count for Broken Links
+  addCheckboxCountListeners("[data-poi-list-checkbox]", "[data-broken-links-list-selection-count]");
```

Comment:
If I understand correctly, the counting process is exactly identical for all lists, right?
I think we won't have more than one list with counter on one page in the near future (this doesn't even work with our current bulk action implementation), so probably you can just use the existing class `bulk-select-item` to count the checkboxes and one hardcoded id `list-selection-count` or similar for the result, then you don't have to call this function with different queryselectors...